packages/@aws-cdk/assert/lib/expect.ts

```diff
@@ -20,7 +20,7 @@ export function expect(stack: api.SynthesizedStack | cdk.Stack, skipValidation =
 
     sstack = {
       name: stack.name,
-      template: stack.toCloudFormation(),
+      template: stack._toCloudFormation(),
```

Comment:
Why we renaming to `_toCloudFormation` given that we have awslabs/jsii#390 in the pipes? Can we not just `@internal` the method?

Reply:
I think in the CDK I'd like to standardize (awslint) that internal members are also prefixed with an underscore to make it clear to maintainers that they are internal.